### ISSUE TITLE
[a11y] Cannot create new Institutional roles based on template/existing roles

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1512]
I cannot create a new Institutional role based on an existing one. Collection roles can be made correctly.

![gif](https://user-images.githubusercontent.com/37256050/167711125-de043376-b127-4e8c-a7d4-860d63421722.gif)

Windows 10, Edge
